### ISSUE TITLE
Scroll is broken after zoom minimumZoomScale

### ISSUE THREAD

**Issue by ch3rn1k** [intergalacticspacehighway/react-native-reanimated-zoom#13]
Hello! Can't scroll inside ScrollView after zooming to minimum with `value < 1`. Any idea how to fix that?

**Comment by intergalacticspacehighway** [comment]
Hi @ch3rn1k Can you share a code snippet? Also, a screen record would be helpful to debug!

Ideally, zoom should reset to 1 if you scroll below 1. Reset is being done [here](https://github.com/intergalacticspacehighway/react-native-reanimated-zoom/blob/65d817a77cb146c54a64b4633a71f64a14aa8753/src/zoom.tsx#L174)

**Comment by ch3rn1k** [comment]
Well, it looks like ScrollView stops working after I **zoom in** and after **zoom out** when `minimumZoomScale < 1`...

```jsx
...
import { Zoom, createZoomListComponent } from 'react-native-reanimated-zoom';

const ZoomScrollView = createZoomListComponent(ScrollView);

const Gallery = () => {
  return (
    <View style={styles.container}>
      <ZoomScrollView
        directionalLockEnabled={true}
        horizontal={true}
        showsHorizontalScrollIndicator={false}
        decelerationRate={0}
        pagingEnabled={true}
        disableIntervalMomentum={true}
        snapToEnd={true}
        snapToInterval={Dimensions.get('window').width + defaultMargin / 2}
        contentOffset={{ x: getContentOffset(route.params.index), y: 0 }}
        contentContainerStyle={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}
      >
        <Zoom
            minimumZoomScale={0.5}
            maximumZoomScale={2.5}
            style={[styles.slide, getAspectSize(v.width, v.height), i === 0 && { marginLeft: 0 }]}
            key={i}
          >
            <Animated.View style={[styles.slideHash, { opacity }]}>
              <Blurhash style={styles.slideHash} blurhash={v.blurhash} />
            </Animated.View>
            <Image
              style={[styles.slide, getAspectSize(v.width, v.height), i === 0 && { marginLeft: 0 }]}
              source={{ uri: v.value }}
              onLoadEnd={handleOnLoad}
            />
          </Zoom>
      </ZoomScrollView>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: 'black'
  },
  slide: {
    position: 'relative',
    width: Dimensions.get('window').width,
    marginLeft: defaultMargin / 2,
    backgroundColor: 'black',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center'
  },
  slideHash: {
    position: 'absolute',
    top: 0,
    right: 0,
    bottom: 0,
    left: 0,
    zIndex: 5
  }
});
```

**Comment by intergalacticspacehighway** [comment]
Thanks for repro! @ch3rn1k. Should be fixed in 0.3.1. Let me know if it doesn't work!

**Comment by ch3rn1k** [comment]
> Thanks for repro! @ch3rn1k. Should be fixed in 0.3.1. Let me know if it doesn't work!

Yeah, works fine now, thanks
